testing/nqp/APKBUILD

```diff
@@ -15,11 +15,6 @@ subpackages="$pkgname-doc"
 source="${pkgname}-${pkgver}.tar.gz::https://github.com/perl6/nqp/archive/${pkgver}.tar.gz"
 builddir="$srcdir"/"$pkgname"-"$pkgver"
 
-# FIXME: some tests fail on armhf and x86, need to investigate more
-case "$CARCH" in
-	armhf | x86) options="$options !check";;
```

Comment:
does it builds now?

Reply:
It does on my host.  It does on Travis-CI.  Will it on the alpine build host?  I don't know.

Reply:
I mean comment tells that on this arches some tests fails, I have only x86_64 and arm8 so can't tests